src/test/common/ios/plistBuddy.test.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {PlistBuddy} from "../../../common/ios/plistBuddy";
+
+import * as Q from "q";
+import * as sinon from "sinon";
+
+suite("plistBuddy", function() {
+    suite("commonContext", function() {
+        test("should attempt to modify, then add, plist properties", function() {
+            const plistFileName = "testFile.plist";
+            const plistProperty = "myProperty";
+            const plistValue = "myValue";
+
+            const mockedExecFunc = sinon.stub();

```

Comment:
Based on this: http://sinonjs.org/docs/ it makes me wonder if this should be a mock instead of a stub:
When to use mocks?

Mocks should only be used for the method under test. In every unit test, there should be one unit under test. If you want to control how your unit is being used and like stating expectations upfront (as opposed to asserting after the fact), use a mock.

When to not use mocks?

Mocks come with built-in expectations that may fail your test. Thus, they enforce implementation details. The rule of thumb is: if you wouldn’t add an assertion for some specific call, don’t mock it. Use a stub instead. In general you should never have more than one mock (possibly with several expectations) in a single test.